app/Models/Topic.php

```diff
@@ -114,12 +115,22 @@ public function scopeVisible($query)
         return $query->where('is_visible', 1);
     }
 
+    public function scopeListing($query)
+    {
+        return $query->visible()->orderBy('name', 'asc')->get()->groupBy('category');
+    }
+
     /*
     |--------------------------------------------------------------------------
     | ACCESORS
     |--------------------------------------------------------------------------
     */
 
+    public function getRecentAttribute()
+    {
+        $dt = Carbon::now();
+        return ceil($dt->diffInDays($this->updated_at) / 5);
```

Comment:
This could be called `getDaysSinceUpdated` and go like this
```suggestion
        return $dt->diffInDays($this->updated_at);
```

Then in the template, you could go 
```blade
{{$topic->days_since_updated < 15 ? 'recent' : ''}}{{$topic->days_since_updated < 5 ? 'recent-1' : ''}}
```
etc